### DISCUSSION TITLE
Tags/labels for Plugin installation (pm install)

### DISCUSSION THREAD

**Discussion by Garulf** [Flow-Launcher/Flow.Launcher#671] [Ideas]
Im wondering if it would be a good idea to have tags/labels for plugins. The plugin dev would need to provide optional tags when submitting to the manifest. `pm install` will return matching tags as well.

This allows a user to search for plugins that relate to a specific category.

**Comment by deefrawley** [comment]
I think it's a great idea

**Reply by jjw24** [reply]
agree

**Comment by Garulf** [comment]
Getting familiar with the plugin manager plugin. I'll look into setting this up soon.
